### ISSUE TITLE
相似度计算实测下来完全不对

### ISSUE THREAD

**Issue by yumianhuli1** [OFA-Sys/Chinese-CLIP#255]
比如文字和图片完全不搭边，相似度得分也会很高，高于0.4、0.45。。。
Why？
thanks

**Comment by hermanWu55** [comment]
是的 chinese clip利用的wukong数据集是新闻图集，图文对 中，文本的描述非常详尽。如果说使用的时候，文本很简单 或者不像他们的文本一样。那么区分就不会很明显了。相对正负样本 得分都差不多

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
